src/crafting.cpp

```diff
@@ -483,12 +515,17 @@ int recipe::batch_time(int batch) const
     } else if (assistants >= 2) {
         total_time = total_time * .60;
     }
-    if (total_time < time)
-        total_time = time;
+    if (total_time < local_time)
+        total_time = local_time;
 
     return (int)total_time;
 }
 
+bool recipe::has_flag(const std::string &flag_name) const
+{
+    return flags.count(flag_name);
```

Comment:
`.count()` is the appropriate way to check for membership in a `std::set`, oddly enough.